NOTICES

```diff
@@ -566,6 +566,7 @@ haskell TLS 1.6.6 : BSD 3-clause "New" or "Revised" License
 haskell TLS 1.6.7 : BSD 3-clause "New" or "Revised" License
 haskell TLS 1.7.5 : BSD 3-clause "New" or "Revised" License
 haskell-adjunctions 4.4 : BSD 2-clause "Simplified" License
+haskell-conduit 0.8.0 : BSD 3-clause "New" or "Revised" License
```

Comment:
this is just wrong, we don’t depend on 4 different versions of conduit. We only depend on conduit-1.3.4. In fact, I’m now not sure if anybody looked at the notices file. There seems to be tons of duplicated Haskell libs in different versions and all of them are wrong. @sormeter do you know what’s up with this? It looks like blackduck performs its own version resolution or something like that which makes no sense.